python/test/unit/cuda/test_experimental_tma.py

```diff
@@ -429,11 +432,15 @@ def matmul_kernel_make_tensor_desciptor(a_ptr, b_ptr, c_ptr,  #
 
 
 @requires_tma
+@pytest.mark.interpreter
 @pytest.mark.parametrize("num_stages", [1, 4])
 @pytest.mark.parametrize("BLOCK_M, BLOCK_N, BLOCK_K", [(32, 32, 32), (128, 64, 64), (128, 128, 64), (128, 256, 64)])
 def test_experimental_make_tensor_descriptor_matmul(num_stages, BLOCK_M, BLOCK_N, BLOCK_K):
     device = "cuda"
-    M, N, K = 8192, 8192, 1024
+    if is_interpreter():
+        M, N, K = BLOCK_M, BLOCK_N, BLOCK_K
+    else:
+        M, N, K = 8192, 8192, 1024
```

Comment:
Reduce the input sizes brings the test time down under 1s but still giving an end-to-end matmul test.

Reply:
Sounds good